apps/web/src/pages/dao/[token]/[tokenId].tsx

```diff
@@ -109,9 +111,8 @@ const TokenPage: NextPageWithLayout<TokenPageProps> = ({
       <Auction auctionAddress={addresses.auction} collection={collection} token={token} />
       <SectionHandler
         sections={sections}
-        activeTab={query?.tab ? (query.tab as string) : undefined}
-        collectionAddress={collection}
-        tokenId={tokenId}
+        activeTab={activeTab}
+        basePath={tokenId ? `/dao/${collection}/${tokenId}` : `/dao/${collection}`}
```

Comment:
I think for this page tokenId should always be defined no ? So this can just be `/dao/${collection}/${tokenId}`